plugins/jira-dashboard/src/annotations.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export const PROJECT_KEY_ANNOTATION = 'jira/project-key';
+export const COMPONENT_ANNOTATION = 'jira/component';
+export const FILTER_ANNOTATION = 'jira/filter-ids';
```

Comment:
I noticed these are repeated in both the frontend and backend, this would be a great opportunity to create a `jira-dashboard-common` package for these, there are several examples of this in the code base if you are unsure and you can create a common package using the `yarn new` and picking "plugin-common"